typewise_alert.test.py

```diff
@@ -1,10 +1,11 @@
 import unittest
-import typewise_alert
+from breach.check_and_alert import Check_and_Alert
 
+test_obj = Check_and_Alert()
 
 class TypewiseTest(unittest.TestCase):
   def test_infers_breach_as_per_limits(self):
-    self.assertTrue(typewise_alert.infer_breach(20, 50, 100) == 'TOO_LOW')
+    self.assertTrue(test_obj.typewise_alert('PASSIVE', 'TO_CONTROLLER', 'rahul', ['temperature'], [25]) == '65261, NORMAL')
```

Comment:
Good abstractions are used 👍 
Additional test cases are needed to obtain good coverage.